pyne/__init__.py

```diff
@@ -2,15 +2,17 @@
 from warnings import warn
 
 
-__version__ = '0.7.1'
 
 if os.name == 'nt':
     p = os.environ['PATH'].split(';')
     lib = os.path.join(os.path.split(__file__)[0], 'lib')
     os.environ['PATH'] = ";".join([lib] + p)
 
 try:
+    from .pyne_version import PYNE_VERSION
```

Comment:
If we've set the `version` keyword in the `setup` call when installing PyNE, why do we need these lines?

Reply:
I don't know, I just kept everything as it was

Reply:
Hrmm I see what you mean. That's odd to me though. Gonna try this out locally....